packages/fxa-shared/metrics/glean/fxa-ui-metrics.yaml

```diff
@@ -1501,6 +1503,42 @@ cad_mobile_pair_use_app:
     data_sensitivity:
       - interaction
 
+cad_redirect_desktop:
+  view:
+    type: event
+    description: |
+      User viewed the "Switch to Firefox" page state after trying to access the pair flow
```

Comment:
While probably technically correct, the inclusion of 'state' makes this a little awkward to read.